src/query/config/src/inner.rs

```diff
@@ -382,3 +342,49 @@ impl Debug for MetaConfig {
             .finish()
     }
 }
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum CatalogConfig {
+    Hive(CatalogHiveConfig),
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum ThriftProtocol {
+    Binary,
+    // Compact,
+}
+
+impl FromStr for ThriftProtocol {
```

Comment:
I thought we would never use this.

Reply:
> I thought we would never use this.

It's used here in line 412:
https://github.com/ClSlaid/databend/blob/e17bfcb78f6887a699601ca6c1311417a6833fcf/src/query/config/src/outer_v0.rs#L407-L415

Reply:
You are right! :heart: